api/build.gradle

```diff
@@ -16,11 +16,9 @@ dependencies {
             libraries.jsr305,
             libraries.animalsniffer_annotations
     compile (libraries.guava) {
-        // prefer 2.3.3 from libraries instead of 2.1.3
+        // prefer our own versions from libraries instead of Guava's dependency
         exclude group: 'com.google.errorprone', module: 'error_prone_annotations'
-        // prefer 3.0.2 from libraries instead of 3.0.1
```

Comment:
This comment has been outdated; [Guava 28.1](https://search.maven.org/artifact/com.google.guava/guava-parent/28.1-jre/pom) uses `jsr305:3.0.2`.